.eslintrc.json

```diff
@@ -4,6 +4,7 @@
     "es6": true,
     "jest": true
   },
+  "parser": "babel-eslint",
```

Comment:
Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/react-redux/.eslintrc.json)
![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/